src/actions/pipelines.js

```diff
@@ -136,8 +135,7 @@ export function loadPipelineData(pipelineID) {
         active: pipelineID,
       });
 
-      const shouldExpandAllPipelines =
-        display.expandAllPipelines || expandAllPipelines;
+      const shouldExpandAllPipelines = expandAllPipelines;
```

Comment:
I saw you already addressed this in the child branch. Please look for this before we merge this to main

Reply:
I will not merge this branch to main but individual branch, this is just to see all code together. Let me merge above changes to current branch.

Reply:
Just a suggestion, It would be better to merge the combined branch to main instead of individual branches. Like having child branches merge to this and then this branch to main. That way we can be sure our tests are passing in this branch and will also pass in main.